app/controllers/users_controller.rb

```diff
@@ -1,3 +1,36 @@
 class UsersController < ApplicationController
   def new; end
+  
+  def show
+    @user = User.find_by id: params[:id]
```

Comment:
use method `before_action` to call load_user method

Reply:
I try to resolve but i'm not sure about this